en/publish/publish-oppo-instant-games.md

```diff
@@ -22,19 +22,19 @@ Starting with __v2.0.5__, __Cocos Creator__ officially supports the release of g
 
 - Determine whether you need to install [Debugging Tools](http://cdofs.oppomobile.com/cdo-activity/static/quickgame/tools//6de9d1c3f06030ae7c52f5105f60383f.zip) according to your own development needs.
 
-## Release process
+## Release Process
 
 1. Use __Cocos Creator__ to open the project project that needs to be released. Select **OPPO Mini Game** in the **Publishing Platform** dropdown of the **Build Release** panel.
```

Comment:
Maybe it's enough for us to need one "project" here?

**Build Release** panel -> **Build...** panel
**Publishing Platform** -> **Platform**